src/ensembl/src/content/app/species/components/species-app-bar/SpeciesAppBar.tsx

```diff
@@ -14,19 +14,57 @@
  * limitations under the License.
  */
 
-import React from 'react';
+import React, { memo, useMemo } from 'react';
+import { connect } from 'react-redux';
+import isEqual from 'lodash/isEqual';
+
+import { AppName } from 'src/global/globalConfig';
+
+import { getActiveGenomeId } from 'src/content/app/species/state/general/speciesGeneralSelectors';
+import { getEnabledCommittedSpecies } from 'src/content/app/species-selector/state/speciesSelectorSelectors';
 
 import AppBar from 'src/shared/components/app-bar/AppBar';
+import { SelectedSpecies } from 'src/shared/components/selected-species';
+import SpeciesTabsWrapper from 'src/shared/components/species-tabs-wrapper/SpeciesTabsWrapper';
+
+import { RootState } from 'src/store';
+import { CommittedItem } from 'src/content/app/species-selector/types/species-search';
 import { HelpPopupButton } from 'src/shared/components/help-popup';
 
-const SpeciesAppBar = () => {
+type SpeciesAppBarProps = {
+  species: CommittedItem[];
+  activeGenomeId: string | null;
+  onSpeciesSelect: (genomeId: string) => void;
+};
+
+const SpeciesAppBar = (props: SpeciesAppBarProps) => {
+  const speciesTabs = useMemo(() => {
+    return props.species.map((species, index) => (
+      <SelectedSpecies
+        key={index}
+        species={species}
+        isActive={species.genome_id === props.activeGenomeId}
+        onClick={() => props.onSpeciesSelect(species.genome_id)}
+      />
+    ));
+  }, [props.species]);
```

Comment:
Not sure what we are buying with this `useMemo`. `props.species` are objects, so every re-render they will not be equal to each other, so useMemo will re-run the wrapped function.

Reply:
Removed it